fake.py

```diff
@@ -445,7 +465,8 @@ def main():
         foreground_mask_image=findFile(args.foreground_mask_image, args.image_folder),
         webcam_path=args.webcam_path,
         v4l2loopback_path=args.v4l2loopback_path,
-        ondemand=not args.no_ondemand)
+        ondemand=not args.no_ondemand,
+        background_mask_update_speed=getPercentage(args.background_mask_update_speed))
```

Comment:
I would like a flag to completely turn this off.

Reply:
just set it to 100

Reply:
Line 276 still runs if you set it to 100. This extra processing can drop FPS on low end machines.

Reply:
done